static/tabs/instructions.html

```diff
@@ -0,0 +1,45 @@
+<div class="tab-pane fade" id="instructions" role="tabpanel" aria-labelledby="instructions-tab">
+    <div class="container mb-4">
+        <div class="row g-3">
+            <div class="col-lg-6">
+                <h2>Connecting PrusaSlicer/SuperSlicer</h2>
+                <ol class="list-group list-group-numbered mb-3">
+                    <li class="list-group-item">Go to "Printer settings" tab</li>
+                    <li class="list-group-item">Click "Gear"⚙️ (Add/Edit Physical Printer)</li>
```

Comment:
Since we have shifted to using BS icons for everything else, maybe we should use one here for the gear.

Reply:
Done :white_check_mark: